models/repos.js

```diff
@@ -11,6 +11,11 @@ const assert = require('assert')
 const mkdirp = require('mkdirp')
 const open = require('open')
 const path = require('path')
+const mock = require('../tests/mocks')
+
+if (process.env.RUNNING_IN_SPECTRON) {
+  mock(dialog)
+}
```

Comment:
Requiring a file from the tests dir in lib code feels weird, we might for example not want to ship the tests dir with the production bundle. Also, someone could introduce a dependency in the `tests/mocks` file but only add it to `devDependencies`.

Do you maybe see another way to resolve this? Maybe even with a lib that patches `require()`?